english/breakfast/cereal/cereal.md

```diff
@@ -0,0 +1,3 @@
+You wake in the morning, and you are hungry for cereal. One would eat....
+	-[Lucky Charms!] (https://github.com/TheBlurrr/create-your-own-adventure/blob/test-branch/english/breakfast/food/lucky_charms.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.